src/index.css

```diff
@@ -1,3 +1,29 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+@layer utilities {
+  /* scrollbar-container */
+  .scrollbar-container {
+    border: solid 3px transparent;
+    scrollbar-gutter: stable both-edges;
+    /* prevent layout shift while vertical scrolling */
+  }
+
+  /* scrollbar */
+  .scrollbar-container::-webkit-scrollbar {
+    width: 8px;
+    height: 8px;
+  }
+
+  /* scrollbar-track */
+  .scrollbar-container::-webkit-scrollbar-track {
+    background: transparent;
+  }
+
+  /* scrollbar-thumb */
+  .scrollbar-container::-webkit-scrollbar-thumb {
+    background: #d3d3d3;
+    border-radius: 5px;
```

Comment:
@Kashyap2701 There seems to be a lot of "magic" numbers and hex colors here. 
Per the design system guidelines, should we rather try to have variables for color/spacing/measurements etc? 

@anjali-sim as FYI since she has worked initially on design system